pages/resources/uas.jsx

```diff
@@ -12,6 +12,7 @@ export default function UAS() {
       <PageLayout>
         <Title title="University Admissions Server (UAS)" arrowLink="/resources" />
```

Comment:
Arrow placement at smaller screen widths is awkward.
![image](https://user-images.githubusercontent.com/46069730/128874079-453f3131-d22c-4376-a9e9-63f87681908d.png)
See my other comment for another example of the arrow being kind of weird in the titlebar. (I forgot to put that comment under my review haha)

Reply:
changes to the `Title` component would probably be made on a separate branch and PR so probably don't worry about this for now - just something to keep in mind

Reply:
Yeah, I'll make a ticket to `fix title back arrow` and look into it later :+1: